src/ConfigAndContainerTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-tooling for the canonical source repository
+ * @copyright Copyright (c) 2018-2019 Zend Technologies USA Inc. (https://www.zend.com)
```

Comment:
Only need the start year here (2019), since it's a new file. (We add the range when we update the file.)